### PR TITLE
Add configurable min-lot and tick size to State

### PR DESCRIPTION
**Pull request by filipzeta**

### CODE REVIEW

File: src/utils.ts [hunk 1]
```diff
@@ -51,6 +51,41 @@ import cloneDeep from "lodash.clonedeep";
...
 
+export function getNativeTickSize(asset: Asset): number {
+  return 100;
+}
+
+export function getDecimalMinLotSize(asset: Asset): number {
+  return getNativeMinLotSize(asset) / 10 ** constants.POSITION_PRECISION;
+}
+
+export function getNativeMinLotSize(asset: Asset): number {
```
Comment (MeanMangosteen) [inline src/utils.ts:62]: Curious why this isn't an a record in constants vs a getter acting like a record.

Comment (kelvinlau20100) [reply src/utils.ts:62]: because it will require accessing the Exchange object in the future so it can't be static if that makes sense

**Review by xjcaa — APPROVED** [review]

File: src/utils.ts (hunk 1 above)
Comment (MeanMangosteen) [reply src/utils.ts:62]: I see, makes sense.

File: src/utils.ts [hunk 2]
```diff
@@ -51,6 +51,37 @@ import cloneDeep from "lodash.clonedeep";
 import * as os from "os";
 import { OpenOrders, _OPEN_ORDERS_LAYOUT_V2 } from "./serum/market";
 
+export function getNativeTickSize(asset: Asset): number {
+  return Exchange.state.tickSizes[assets.assetToIndex(asset)];
+}
+
+export function getDecimalMinLotSize(asset: Asset): number {
+  return getNativeMinLotSize(asset) / 10 ** constants.POSITION_PRECISION;
```
Comment (MeanMangosteen) [inline src/utils.ts:59]: could use `convertNativeLotSizeToDecimal`.

**Pull request merged**